### ISSUE TITLE
The Slime Elytra are not compatible with Capes

### ISSUE THREAD

**Issue by ManyullynSword** [SlimeKnights/TinkersConstruct#5083]
### Minecraft Version

1.18.2

### Forge Version

40.2.1

### Mantle Version

1.9.43

### Tinkers' Construct Version

3.6.2.108

### Describe your issue

I actually discovered this Bug back in the first 1.18.2 version of Tinkers' Construct. Vanilla's Capes transform into elytra when equipped with vanilla Elytra (disappear without Elytra). When equipped with the Slime Elytra, Capes neither disappeared nor changed into Elytra form, but remained on player's back.

No Elytra
![No](https://user-images.githubusercontent.com/110178522/218315677-1635775f-495c-4e16-84a3-8338ff997a8e.png)

Vanilla Elytra
![Vanilla](https://user-images.githubusercontent.com/110178522/218315767-6156d912-d9bf-4bb1-b2a4-6e590d57cd06.png)

Slime Elytra
![TConstrucct](https://user-images.githubusercontent.com/110178522/218315748-399c86a3-81d0-490a-a4f9-5cbc7942367c.png)

### Crash Report

No response

### Other mods

None

### Tried reproducing with just Tinkers?

Yes

### Performance Enchancers

None of the above

### Searched for known issues?

Checked pinned issues

**Comment by KnightMiner** [comment]
That is correct. While I could give it cape textures, that would mess with our own embellishment system. Pretty sure there is nothing I can do to hide capes.

**Comment by ManyullynSword** [comment]
You can hide Cape directly, which is at least better than models interneathery.

**Comment by ManyullynSword** [comment]
You can hide Cape directly.

**Comment by KnightMiner** [comment]
Yes, the player can hide capes. That does not mean I can hide capes automatically when you put on a slimelytra

**Comment by ManyullynSword** [comment]
You mean... You can't hide?

**Comment by ManyullynSword** [comment]
Letting players hide Cape on their own is obviously a bit of a hassle.You can give the Slimelytra cape texture first and then mask it (personal idea only).

**Comment by KnightMiner** [comment]
Giving slimelytra the cape texture will not hide the cape automatically. The problem still exists.

**Comment by ManyullynSword** [comment]
Change the player's Cape material to blank when the player equips Slimelytra so that it is not displayed (personal thought).

**Comment by KnightMiner** [comment]
Unless you have code snippets to explain what you mean, that sort of suggestion is not useful. I have no clue what you mean by "change the cape material", I think you don't understand how entity renderers work.

**Comment by ManyullynSword** [comment]
Not "material", but "texture". You can draw a blank Cape and change the player's Cape to that blank Cape when the player wears Slimelytra. As for how entity renderers work, I don't really know.

**Comment by KnightMiner** [comment]
I think you are entirely missing my point. **I do not draw the cape**. Mojang draws the cape, and I **cannot** change how that works without huge hacks (if not at all).

Unless youu actually know how entity rendering works, the suggestion is not helpful.
